Inspicio/Controllers/ImagesController.cs

```diff
@@ -270,59 +271,127 @@ public class RatingBody
             int id = data.ImageID;
             var image = await _context.Images.SingleOrDefaultAsync(m => m.ImageID == id);
 
-            // if the like button has been pressed
-            if (data.boolean)
+            // if the up thumb has been pressed
+            if (data.ThumbVal == 1)
```

Comment:
I found this part tricky to follow & keep in my head the various states, it might be nicer to simplify this a bit. If there was an enum for the Approved, Rejected,  ChangesRequested & undecided state (rather than a set of bools) then there would be less code needed so it should be a bit easier to follow.

[[++++__]](https://gist.github.com/WilkaH/29df175966a76687f846)